src/routes.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
Here we are creating a file that contains all the routes and a redirect that redirects / to /home. That <Redirect> means that when you access the site on `localhost:3000` or www.yoursite.com (which is the slash because there is no path after the .com) it will actually be loading the home page because it redirects to /home. 
Note how I am importing the components and setting them to each <Route> component using `component={ComponentName};` then note how we are using the <Routes> component defined in this file, check `src/components/App.jsx` 

In the `App.jsx` file We have the Navigation and the `<Routes>` which is a component we define in this routes.js file, which contains all of our app routes, wrapped by the <Router> which is the `import { BrowserRouter as Router } from 'react-router-dom';` we are setting an alias for the BrowserRouter and calling it just `Router`. I recommend studying about the React Router because it is one of the most important building blocks of React.

You might ask yourself why did I add the `<Navigation>` there? Because the navigation is at the App level and it will appear in any route we navigate to, as it is alongside the <Routes>, and remember Routes it going to take care of loading our components based on wich Route we load.

```
return (
    <Router> // React Router Dom
      <Navigation /> // navigation component that will be visible in any page
      <Routes /> // Our app defined routes
    </Router>
  );
```